resources/lang/cn.js

```diff
@@ -84,20 +84,40 @@ class CactbotLanguageCn extends CactbotLanguage {
       Pull: '开怪',
     });
 
+    this.kPetNames = Object.freeze([
+      // Pulled from http://cafemaker.wakingsands.com/Pet?pretty=true
+      '绿宝石兽',
+      '黄宝石兽',
+      '伊弗利特之灵',
+      '泰坦之灵',
+      '迦楼罗之灵',
+      '朝日小仙女',
+      '夕月小仙女',
+      '车式浮空炮塔',
+      '象式浮空炮塔',
+      '亚灵神巴哈姆特',
+      '亚灵神不死鸟',
+      '炽天使',
+      '月长宝石兽',
+      '英雄的掠影',
+      '后式自走人偶',
+      '分身',
+    ]);
+
     this.countdownStartRegex = function() {
-      return Regexes.parse(/^.{14} 00:.{4}:距离战斗开始还有(\y{Float})秒！/);
+      return Regexes.parse(/^(?<timestamp>\y{Timestamp}) 00:.{4}:距离战斗开始还有(?<time>\y{Float})秒！/);
```

Comment:
Can you use `gameLog` for these as well?

Reply:
I can modify the existing logic to use gameLog instead, yes. This was just to add the capturing groups, and I'm not sure how exactly that'll work for these (e.g. can I give a capturing group in the gameLog `line` parameter and have it capture?)

Reply:
Yeah, adding capturing groups should work just fine.

Reply:
Huh. Today I learned that nested capturing groups work in JS. Alright, I'll go through and adjust all of these for all of the languages.

Reply:
Hmm, I get the feeling most of the other language regexes didn't match the entire line (missing ending punctuation, or in the case of FR's countdownCancelRegex, areaSealRegex, and areaUnsealRegex missing a bunch of stuff). Are there any full examples of these lines for all languages?

Until the regexes can be modified to match their full line, I have made sure they end with `.*?` as a catch-all when they don't visibly already end with some form of punctuation.

Reply:
Unfortunately, no.  Everything was pretty sloppy about matching the full line previously, sorry.

Reply:
Alright. Just to be safe I'll make a last pass through and put `.*?` both prefixing and suffixing the line match for anything that wasn't preceeded by a `:`.